lib/modules/apostrophe-schemas/index.js

```diff
@@ -802,7 +802,8 @@ module.exports = {
       toGroups: function(fields) {
         return self.toGroups(fields);
       },
-      field: function(field) {
+      field: function(field, readOnly = false) {
```

Comment:
Default values are an ES6-ism, please remove. Apostrophe 2.x core is compatible back to ES5.